recipes/openblas/all/conanfile.py

```diff
@@ -39,17 +39,20 @@ def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+        if self.options.build_lapack:
+            self.output.warn("Building with lapack support requires a Fortran compiler.")
```

Comment:
I think the previous location of this warning was correct.
A lapack compiler is only required when building openblas, not when using it.